openstates/models/committees.py

```diff
@@ -45,7 +45,7 @@ def validate_parent_and_classification(
     ) -> dict[str, typing.Any]:
         if (
             values.get("classification") == "subcommittee"
-            and values.get("parent") in COMMITTEE_PARENTS
+            and values.get("parent") not in COMMITTEE_PARENTS
```

Comment:
in retrospect this code was correct, I misunderstood where it was

committees should have a COMMITTEE_PARENT but subcommittee parents should be the name of the parent :)